### ISSUE TITLE
Support job delete

### ISSUE THREAD

**Issue by clausmichele** [Open-EO/openeo-aggregator#16]
Currently it is not possible to delete a job with the aggregator but it is a feature we would need sooner or later.

**Comment by soxofaan** [comment]
What error do you get if you try?

Because both job canceling and job deletion are already implemented.
Covered by these tests:

cancel:
https://github.com/Open-EO/openeo-aggregator/blob/a709bf140943d62dac047a60f40de1c4d42eb393/tests/test_views.py#L511-L515

delete:
https://github.com/Open-EO/openeo-aggregator/blob/a709bf140943d62dac047a60f40de1c4d42eb393/tests/test_views.py#L533-L537

**Comment by clausmichele** [comment]
You are right, tested and it works. @m-mohr please see here, deleting a job is already supported by the aggregator and hence can also offered in the Web Editor,

**Comment by m-mohr** [comment]
Might be that there is a hidden functionality, but it's not exposed in the capabilities and as such the Hub, Editor etc. won't allow it.
